BUILD.bazel

```diff
@@ -0,0 +1,49 @@
+load("@build_bazel_rules_nodejs//:defs.bzl", "npm_package", "rollup_bundle")
+load("@npm_bazel_jasmine//:index.bzl", "jasmine_node_test")
+load("@npm_bazel_typescript//:defs.bzl", "ts_library")
+load("//lib:bundle.bzl", "ES2015_BUNDLES", "ES5_BUNDLES", "ES5_GLOBAL_BUNDLES")
+
+exports_files([
+    "tsconfig.json",
+    "tsconfig-node.json",
+    "angular_license.txt",
+])
+
+npm_package(
+    name = "npm_package",
+    srcs = [
+        "CHANGELOG.md",
+        "LICENSE",
+        "LICENSE.wrapped",
+        "README.md",
+        "package.json",
+    ],
+    replacements = {
+        # Since the bazel package is experimental, patch the version number
```

Comment:
this seems like copy-paste - this isn't the bazel package

Reply:
You are right, I removed this `replacements`.